projects/plugins/jetpack/extensions/blocks/sharing-button/components/view-button.jsx

```diff
@@ -0,0 +1,24 @@
+import SocialIcon from 'social-logos';
+
+const ViewSocialButton = ( { service, url } ) => {
+	return (
+		<a
```

Comment:
From an accessibility perspective, we could make screen readers read something like `Share to Facebook` rather than `Facebook`. This would provide more context to screen reader users and avoid confusion if there are multiple link labelled `Facebook` on the page.

We could do that by adding an `aria-label` attribute on the `a` and, I believe, hiding the `span` with `aria-hidden="true"`.